ush/ufsda/archive.py

```diff
@@ -35,16 +35,26 @@ def atm_diags(config):
         r2d2_config['source_file_fmt'] = input_file.replace('.nc4', '_0000.nc4')
         r2d2_config['obs_types'] = [ob['obs space']['name']]
         ufsda.r2d2.store(r2d2_config)
-        # get bias files if needed
+        # store bias files
         if 'obs bias' in ob.keys():
             r2d2_config['type'] = 'bc'
             r2d2_config['provider'] = 'gsi'
             r2d2_config['start'] = config['valid_time']
             r2d2_config['end'] = config['valid_time']
+
+            # store satbias
             r2d2_config['file_type'] = 'satbias'
             target_file = ob['obs bias']['output file']
             r2d2_config['source_file_fmt'] = target_file
             ufsda.r2d2.store(r2d2_config)
+
+            # store satbias_cov
+            r2d2_config['file_type'] = 'satbias_cov'
```

Comment:
Not a comment on this PR but a mental note for me/us, this is 'non-standard' R2D2. I had to manually modify the schema to support this. Need to discuss with @aerorahul if we are going to maintain our own NCEP schema (if we use R2D2).

Reply:
It would be good to get clarity on whether or not we will use R2D2 in operations sooner rather than later.